confmap/README.md

```diff
@@ -83,6 +83,74 @@ The `Resolve` method proceeds in the following steps:
 4. For each "Converter", call "Convert" for the "result".
 5. Return the "result", aka effective, configuration.
 
+#### New Merging strategy
+
+When merging configs from different sources, we currently override static values such as slices, numbers, and strings.
+However, you can combine lists instead of discarding the existing ones by enabling `confmap.enableMergeAppendOption` feature flag. 
```

Comment:
```suggestion
You can opt-in to experimentally combine slices instead of discarding the existing ones by enabling `confmap.enableMergeAppendOption` feature flag. Lists are appended in the order in which they appear in their configuration sources.

This will **not** become the default in the future, we are still deciding how this should be configured and want your feedback on [this issue](https://github.com/open-telemetry/opentelemetry-collector/issues/8754).
```